reportportal_client/service.py

```diff
@@ -27,6 +27,11 @@
 
 from .errors import ResponseError, EntryCreatedError, OperationCompletionError
 
+try:
```

Comment:
Why did not you use the six module? It's just a one-line change.

from six.moves import collections